aws-greengrass-testing-features/aws-greengrass-testing-features-api/src/main/java/com/aws/greengrass/testing/features/CommanConnectivitySteps.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.testing.features;
+
+import com.aws.greengrass.testing.api.device.Device;
+import com.aws.greengrass.testing.api.model.PillboxContext;
+import com.aws.greengrass.testing.platform.Platform;
+import com.aws.greengrass.testing.platform.PlatformResolver;
+import io.cucumber.guice.ScenarioScoped;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.When;
+
+import java.io.IOException;
+import javax.inject.Inject;
+
+/**
+ * Checks the connectivity for platfroms.
+ *
+ * @throws IOException {throws IOException}
+ * @throws InterruptedException {throws IInterruptedException}
+ */
+@ScenarioScoped
+public class CommanConnectivitySteps {
```

Comment:
Seems like we want to delete this file, it is a duplicate there are `CommanConnectivitySteps.java` and `CommonConnectivitySteps.java`

Reply:
resloved